### PR TITLE
Fix misunderstanding in tutorial

### PR DESCRIPTION
**Pull request by dortonway**
According to the topic: https://discuss.reactjs.org/t/tutorial-misunderstanding/2769/2

### CODE REVIEW

File: docs/docs/tutorial.md [hunk 1]
```diff
@@ -416,7 +416,7 @@ var CommentBox = React.createClass({
 });
 ```
 
-Here, `componentDidMount` is a method called automatically by React when a component is rendered. The key to dynamic updates is the call to `this.setState()`. We replace the old array of comments with the new one from the server and the UI automatically updates itself. Because of this reactivity, it is only a minor change to add live updates. We will use simple polling here but you could easily use WebSockets or other technologies.
+Here, `componentDidMount` is a method called automatically by React when a component is rendered for the first time. The key to dynamic updates is the call to `this.setState()`. We replace the old array of comments with the new one from the server and the UI automatically updates itself. Because of this reactivity, it is only a minor change to add live updates. We will use simple polling here but you could easily use WebSockets or other technologies.
```
Comment (jimfb) [inline docs/docs/tutorial.md]: ...automatically by React **after** a component...

"when" is ambiguous (before? during? after?).

**Comment by jimfb** [comment]
One last thing, can you squash the commits together and repush?  Then we can merge.

**Comment by jimfb** [comment]
You didn't need to delete the PR, you could have updated the branch using `git push -f` after squashing.  Either way, whatever works.

**Comment by jimfb** [comment]
Great, thanks!

**Comment by dortonway** [comment]
@jimfb, actually, I don't know why, but after squashing, in commits in the pull request a saw the commit "Capitalizes first letter of first word in the Step 1 instruction" by kchia. So I reseted branch to the commit before mine, pulled changes from react master and commited my changes again.

**Pull request merged**